web_chat.lua

```diff
@@ -95,7 +95,8 @@ local JavaScript = [[
 		
 	</script>
 ]]
-
+-- Plugin name, function name --
+local OnWebChatCallbacks = {}
```

Comment:
The comment could be more specific about the layout of the table: `array of {PluginName, FunctionName} tables`